metadata-db-app/src/cmr/metadata_db/services/concept_service.clj

```diff
@@ -378,10 +377,10 @@
   (let [{:keys [concept-id revision-id]} (save-concept-revision mdb-context concept)]
     {:concept-id concept-id, :revision-id revision-id}))
 
-(defn- delete-association
-  "Delete the association with the given native-id using the given embedded metadata-db context."
-  [mdb-context source-concept-type native-id]
-  (let [source-concept-type-str (name source-concept-type)
+(defn- delete-variable-association
+  "Delete variable association with the given native-id using the given embedded metadata-db context."
+  [mdb-context native-id]
+  (let [source-concept-type-str "variable"
         assoc-concept-type (keyword (str source-concept-type-str "-association"))
```

Comment:
there is no need to define `source-concept-type-str` and you can delete this line as well. Just replace `assoc-concept-type on line 386 with `:variable-association`.

Reply:
Done.